netty-server/docker-compose.yml

```diff
@@ -62,8 +68,23 @@ services:
 
   dynamodb:
     image: amazon/dynamodb-local:1.24.0
-    command: "-jar DynamoDBLocal.jar -sharedDb -dbPath /data"
+    command: "-jar DynamoDBLocal.jar -sharedDb -dbPath ./data"
     ports:
       - "8000:8000"
+    user: root
+    volumes:
+      - ./formkiq/dynamodb:/home/dynamodblocal/data
+    working_dir: /home/dynamodblocal
+
+  keycloak:
+    image: quay.io/keycloak/keycloak:26.0.6
+    command: start-dev --db=dev-file --http-port=8081 --import-realm #place formkiq-realm.json to /formkiq/keycloak/import/
+    #command: export --dir /opt/keycloak/data/import/ --realm formkiq --users realm_file #To export data comment line above and uncomment this line
```

Comment:
Serves as documentation how to re-export settings in case if it'll be needed for customers/developers to debug issues or to share same configuration.